### ISSUE TITLE
Incorrect report of missing attribute when using mixin and explicit subclassing of Protocol

### ISSUE THREAD

**Issue by thomkeh** [microsoft/pyright#3111]
**Describe the bug**
When a Protocol specifies an attribute, and a mixin class provides that attribute, then pyright still reports it as being missing.

**To Reproduce**
```python
from typing import Protocol

class P(Protocol):
    x: int

class Mixin:
    x = 3  # (I also tried annotating this variable but it made no difference)

class B(Mixin, P):  # error here (see below)
    pass

b = B()
print(b.x)
```
Output:
```
Found 1 source file
/home/[...]/test_proto.py
  /home/[...]/test_proto.py:9:7 - error: Class derives from one or more protocol classes but does not implement all required members
    Member "x" is declared in protocol class "P" (reportGeneralTypeIssues)
1 error, 0 warnings, 0 informations
```

**Expected behavior**
No error, because this works fine at run time.

**VS Code extension or command-line**
Command-line with v1.1.222.

**Additional context**
To be honest, I'm not completely sure whether this is worth fixing. I'm not actually using code like this; I just stumbled upon it while trying out something and thought I'd report it.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.225, which I just published. It will also be included in the next release of pylance.
